### PR TITLE
Completed Challenge 13

### PR DESCRIPTION
**Pull request by UreaLaden**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code-challenges/javaChallenges/lib/src/main/java/javaChallenges/utilities/MultiBracketValidation.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+public class MultiBracketValidation {
+
+  public static boolean multiBracketValidation(String input){
+    int braceMatch = 0;
+    int bracketMatch = 0;
+    int parenMatch = 0;
+
+    for(String a: input.split("")){
+      switch(a){
```
Comment (Mattpet26) [inline code-challenges/javaChallenges/lib/src/main/java/javaChallenges/utilities/MultiBracketValidation.java:11]: Interesting approach using a switch with a few ternaries
